rqt_py_common/src/rqt_py_common/message_field_type_helpers.py

```diff
@@ -0,0 +1,397 @@
+# Copyright (c) 2020, Michael Lautman PickNik Robotics
```

Comment:
@brawner This file is the most important one in the PR as most of the changes can be classified as either 1) updating the field_type parsing methods to support bounded and unbounded arrays and 2) Moving field_type parsing functionality that had been spread across message_helpers and topic_helpers into this file.